packages/shared/src/lib/user.ts

```diff
@@ -86,6 +90,9 @@ export async function logout(): Promise<void> {
 }
 
 export async function deleteAccount(): Promise<void> {
+  return await request(`${apiUrl}/graphql`, DELETE_USER_MUTATION);
+  return Promise.resolve();
```

Comment:
Line 93 contains a return statement, so there is no need for the rest, I guess?

Reply:
Totally right, forgot to remove the test data there